lib/package.gd

```diff
@@ -691,6 +691,31 @@ DeclareGlobalFunction( "LoadPackageDocumentation" );
 ##
 ##  <Description>
 ##  loads the &GAP; package with name <A>name</A>.
+## <P/>
+##  As an example, the following loads the &GAP; subgroups package
```

Comment:
> 
> I tried make manuals from the GAP repo root but received the following message
> 
> make: *** No rule to make target `manuals'.  Stop.

Ah, I was assuming that GAP is built already since you need it to build manuals. 
So, the full process from a clean checkout is

./configure
make
make manuals